dist/debian/DEBIAN/control

```diff
@@ -0,0 +1,6 @@
+Package: nebula
+Version: VERSION
+Architecture: ARCHITECTURE
+Maintainer: Freifunk München <hilfe@ffmuc.net>
```

Comment:
I don't have an address I could paste here so I used something at least I am able to respond.
So if you have suggestions for a more appropriate address I can change it.

Reply:
Does "SlackHQ <https://github.com/slackhq/nebula>" work? Or is a email address mandatory?

Reply:
It could be done, I assume, but given the [debian maintainers guide](https://www.debian.org/doc/manuals/maint-guide/dreq.en.html#control) it should contain a valid mail address:
> Line 4 is the name and email address of the maintainer. Make sure that this field includes a valid To header for email, because after you upload it, the bug tracking system will use it to deliver bug emails to you. Avoid using commas, ampersands, or parentheses.